tofhir-engine/src/main/scala/io/tofhir/engine/model/FhirMappingTaskExecutionException.scala

```diff
@@ -0,0 +1,10 @@
+package io.tofhir.engine.model
+
+/**
+ * Exception to be thrown when there is an error during mapping task execution.
+ *
+ * @param msg the message
+ * */
+final case class FhirMappingTaskExecutionException(private val msg: String)
```

Comment:
Up to now we have been using the FhirMappingException class to throw and handle the expcetions throughout the system. We recently added FhirMappingInvalidResourceException class to specifically identify and handle the errors occuring while communicating with the FHIR endpoint. Now we are adding FhirMappingTaskExecutionException to differentiate another specific error during the task execution.

Shall we have the FhirMappingException as the umbrella exception class for the whole system and the others extending this for specific things? Then shall we walk through all places that we throw/handle FhirMappingException to check whether we use extended classes there or not? Maybe we can discuss these in a meeting.

Reply:
I've just removed that class and used FhirMappingException instead 👍